io/EptReader.cpp

```diff
@@ -588,9 +587,15 @@ PointViewSet EptReader::run(PointViewPtr view)
             uint64_t startId(0);
 
             if (m_info->dataType() == EptInfo::DataType::Laszip)
-                startId = readLaszip(*view, key, nodeId);
+            {
+				std::unique_ptr<PointTable> table(nullptr);
+                startId = readLaszip(*view, key, nodeId,table);
+                table.reset(nullptr);
```

Comment:
This extra table should not be necessary

Reply:
Done!! 
Reverted to original.